### PR TITLE
Add script which will delete images from cloudinary which are more than 30 days old

### PR DESCRIPTION
**Pull request by JakeChampion**
This script will be executed by the [Heroku Scheduler Addon](https://devcenter.heroku.com/articles/scheduler) once every day on the production EU Heroku application.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "http-proxy": "1.18.1",
     "jsdom": "16.4.0",
     "lookup-dns-cache": "^2.1.0",
+    "make-promises-safe": "^5.1.0",
```
Comment (notlee) [inline package.json:55]: 👀

Comment (JakeChampion) [reply package.json:55]: lol

**Comment by JakeChampion** [comment]
@notlee This is ready to review now, I just added the sentry integration to it 👍

File: scripts/delete-old-images-from-cloudinary.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		next_cursor = result.next_cursor;
+		const imageIDs = result.resources.map(resource => resource.public_id);
+		yield imageIDs;
+	} while (next_cursor);
+}
+
+/**
+ * Delete images from Cloudinary
+ *
+ * @param {string[]} images an array of strings which are the public IDs of the cloudinary images.
```
Comment (notlee) [inline scripts/delete-old-images-from-cloudinary.js:40]:
```suggestion
 * @param {string[]} an array of strings which are the public IDs of the cloudinary images.
```

File: scripts/delete-old-images-from-cloudinary.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	api_key: process.env.CLOUDINARY_API_KEY,
+	api_secret: process.env.CLOUDINARY_API_SECRET,
+});
+
+/**
+ * Creates an async iterable which contains cloudinary image public ids of images which were uploaded over 30 days ago.
+ *
+ * @returns {AsyncIterable<string[]>} An asynchronous iterable of an array of strings which are the public IDs of the cloudinary images.
+ */
+async function* getAllImagesUploadedMoreThan30DaysAgo() {
```
Comment (notlee) [inline scripts/delete-old-images-from-cloudinary.js:23]: love it

**Review by notlee — APPROVED** [review]

File: scripts/delete-old-images-from-cloudinary.js (hunk 2 above)
Comment (JakeChampion) [reply scripts/delete-old-images-from-cloudinary.js:40]: I thought that the parameter name is required for jsdoc to know which parameter the documenation is intended for, is that not required when there is only one parameter?

Comment (JakeChampion) [reply scripts/delete-old-images-from-cloudinary.js:40]: Yep, can confirm the parameter name is required otherwise JSDoc throws an error.

Comment (JakeChampion) [reply scripts/delete-old-images-from-cloudinary.js:40]: image

**Pull request merged**